documentation-site/components/table-of-contents.js

```diff
@@ -74,6 +74,8 @@ const TableOfContents = props => {
         // to make sure we align vertically with the edit on github button
         marginTop: '-10px',
         marginBottom: 0,
+        // set predictable width to avoid page relayout when table of content changes
+        width: 160
```

Comment:
Linting fails CI because it's expecting a comma at the end of the each object property. Feel free to accept this suggestion in the github UI and it should resolve. Thanks for the bug fix!

```suggestion
        width: 160,
```